packages/protocol-kit/tests/e2e/contractManager.test.ts

```diff
@@ -113,14 +97,14 @@ describe('Safe contracts manager', () => {
           Safe.init({
             provider,
             safeAddress,
-            contractNetworks: customContractNetworks
+            contractNetworks: customContractNetworks as ContractNetworksConfig
```

Comment:
Why do you prefer the cast? If you leave it at definition level will already assist you with the types if you need to modify the object.

Reply:
true, quick change 🙃